langlib/lang.array/src/main/ballerina/array.bal

```diff
@@ -201,6 +264,16 @@ public isolated function setLength((any|error)[] arr, int length) returns () = @
 # Returns `()` if not found.
 # Equality is tested using `==`.
 #
+# ```
+# string[] greetings = ["Hello", "Hola", "Bonjour", "Hola", "Ciao"];
+#
+# // First index of "guten tag" if it exists in the list.
+# greetings.indexOf("guten tag") is () => true
```

Comment:
this could be simplified  to
```suggestion
# greetings.indexOf("guten tag") => ()
```

Reply:
This was to allow for the toString representation, differentiating between nil and empty string. https://github.com/ballerina-platform/ballerina-lang/pull/38989#discussion_r1044052894

Reply:
Is using `toBalString` any better?